lib/android/app/src/test/java/com/reactnativenavigation/views/TitleAndButtonsContainerTest.kt

```diff
@@ -12,7 +12,8 @@ import com.nhaarman.mockitokotlin2.verify
 import com.reactnativenavigation.BaseTest
 import com.reactnativenavigation.options.Alignment
 import com.reactnativenavigation.options.params.Colour
-import com.reactnativenavigation.options.params.NullColor
+import com.reactnativenavigation.options.params.ThemeColour
+import com.reactnativenavigation.options.params.RNNNullColor
```

Comment:
`RNNNullColor` should be `NullThemeColour` right?

Reply:
outdated